llvm_objects.py

```diff
@@ -162,7 +205,7 @@ def _get_instructions(self, l_block, w_module):
             prev = current_instr
             w_instrs.append(current_instr)
             l_instr = llwrap.LLVMGetNextInstruction(l_instr)
-        return w_instrs
+        return w_instrs[:]
```

Comment:
Since `w_instrs` is a fresh list in this function (line 155), I don't see how "it's assigned to an immutable thingy" can be a problem (but maybe I'm missing out on something).